stable/opsportal/values.yaml

```diff
@@ -49,3 +49,55 @@ opsportal:
           serviceName: traefik-kubeaddons-dashboard
           servicePort: 80
         path: /ops/portal/traefik
+
+
+###############################################################################
+#  Konvoy UI
+###############################################################################
+
+image:
```

Comment:
just realized because it's a subchart, don't you need to put all `konvoy-ui` values like such?

```yaml
kommander:
  image:
    repository: mesosphere/kommander
    tag: 1.60.0
    pullPolicy: IfNotPresent
  replicas: 1
# ....

  mode: konvoy
# ....
```

Reply:
example: https://github.com/helm/charts/blob/9fa4db5f3f2981fc56d5e54b9f551a9c86fdfbe0/stable/prometheus-operator/values.yaml#L351

Reply:
but we want to have the konvoy-ui defaults set here

Reply:
```
# i think you need to put `kommander` here because the subchart is "kommader"
# it's basically saying, "apply the values under `kommander` to the `kommander` chart
kommander
  image:
    repository: mesosphere/kommander
    tag: 1.60.0
    pullPolicy: IfNotPresent
  replicas: 1
# ....

  mode: konvoy
# ....
```

in my example:
`grafana` is a subchart: https://github.com/helm/charts/blob/9fa4db5f3f2981fc56d5e54b9f551a9c86fdfbe0/stable/prometheus-operator/requirements.yaml#L13

all the values are under `grafana`: https://github.com/helm/charts/blob/9fa4db5f3f2981fc56d5e54b9f551a9c86fdfbe0/stable/prometheus-operator/values.yaml#L351

i don't know where `enabled` is from in the example, may need to investigate.